iOS/Tests/BaseSDKTest.swift

```diff
@@ -104,6 +83,13 @@ class LoginTests: XCTestCase {
             XCTAssert(title.waitForExistence(timeout: timeout), appLoadError)
+        case .iOS13Swift:
```

Comment:
The iOS 13 swift app has 3 screens:
accounts list -> contacts for selected account -> details for selected contact.
Maybe the test could click through?

Reply:
Just updated this.  I can click through but unfortunately that is about it.  There seems to be a blocker with SwiftUI and XCUITest (I believe because of ListCoreCellHost) where I am unable to get text from the table cells.